Doppler.ContactPolicies.Api/appsettings.Development.json

```diff
@@ -8,5 +8,9 @@
   },
   "DopplerSecurity": {
     "PublicKeysFolder": "public-keys-dev"
+  },
+  "DopplerDataBaseSettings": {
+    "ConnectionString": "Data Source=.;Initial Catalog=ContactPolicies;Integrated Security=True;",
+    "Password": ""
```

Comment:
Did you prepare a local environment? What do you think about adding documentation about how to configure it?

Reply:
This documentation could be part of a different PR.